hub/components/Hub/SideCard/Bounties/index.tsx

```diff
@@ -1,7 +1,124 @@
+import { useEffect, useState } from 'react';
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+import ChevronRightIcon from '@carbon/icons-react/lib/ChevronRight';
+
+import { abbreviateNumber } from '@hub/lib/abbreviateNumber';
+import cx from '@hub/lib/cx';
+
 interface Props {
   className?: string;
+  realmUrlId: string;
+}
+
+interface BountyData {
+  'objectID': string;
+  'Name': string;
+  'type': string;
+  'Opportunity Title': string;
+  'Opportunity Description': string;
+  'Total Value in USD': number;
 }
 
 export function Bounties(props: Props) {
-  return <div className={props.className}></div>;
+  const [bounties, setBounties] = useState([]);
+  const router = useRouter();
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const baseUrl = `https://earn.superteam.workers.dev/?key=${props.realmUrlId.toLowerCase()}`;
+      const res = await fetch(baseUrl, {
+        method: 'GET',
+        headers: {
+          'Accept': 'application/json',
+          'Content-Type': 'application/json',
+        },
+      });
+      const json = await res.json();
+
+      if (res.ok) {
+        try {
+          const listing = JSON.parse(json.data.value);
+          console.log(listing);
```

Comment:
could we remove the console.log?

Reply:
ORCA

Reply:
The console.log should not be there, will clean in up. 

Will also share the company with a listing and add a screenshot for refernce